.github/workflows/main-branch.yml

```diff
@@ -18,10 +18,10 @@ jobs:
   python_cross_platform:
     name: Python Cross Platform Build & Test
     uses: ./.github/workflows/python-cross-platform-support.yml
- 
-# Commenting out from main branch workflow till permissions aspect can be resolved for 
+
+# Commenting out from main branch workflow till permissions aspect can be resolved for
 # reconnecting automated page deployment. The below workflow is able to be ran manually
-# allowing for the site to stay up to date. 
+# allowing for the site to stay up to date.
   # build_and_deploy_documentation:
   #   name: Build and Deploy GitHub Pages
   #   uses: ./.github/workflows/sphinx.yml
```

Comment:
This should be able to be uncommented now since we got the auto deployment back working.

Reply:
We need to correct that in the `.github` repo so that we don't have to update it manually in the others.